Sources/ISDBTibs/TibsBuilder.swift

```diff
@@ -257,6 +257,82 @@ extension TibsBuilder {
     return result
   }
 
+#if os(Windows)
+  private func quoteWindowsCommandLine(_ commandLine: [String]) -> String {
```

Comment:
This and the function that calls it look independent of `self`.  I suggest putting them at the bottom of the file, outside the type.